### DISCUSSION TITLE
Share your weather Photos!

### DISCUSSION THREAD

**Discussion by MET905** [niels747/2D-Weather-Sandbox#32] [General]
I will go first! This is what happens when you have 10 days of sunshine and warm temperatures with an advancing strong cold front! This cold front produced very strong winds ahead and in its wake. followed by heavy rain/Snow. once the front fully passed and wiped out the warm air... a snowstorm began! leaving the area with clearing skies but very chilly. 
![Col front](https://user-images.githubusercontent.com/113155608/206782990-e16466a8-5dad-42f9-8c50-afe562b1e86a.png)

**Comment by Meso22** [comment]
![Desktop 2022 12 10 - 08 59 53 02_Moment](https://user-images.githubusercontent.com/105577122/206809433-cbe1d486-5201-4542-8c47-dfda0d4352dc.jpg)

Here is a massive storm I created from a world I have spent a couple of months working on.

**Comment by Meso22** [comment]
![Desktop 2022 12 10 - 08 59 53 02_Moment(2)](https://user-images.githubusercontent.com/105577122/206809948-d00b0b0e-83f7-498d-9699-e354037d4405.jpg)
![Desktop 2022 12 10 - 08 59 53 02_Moment(4)](https://user-images.githubusercontent.com/105577122/206809963-403d38d2-c3bf-4b4b-bbb4-2fd7652c5a20.jpg)

Here's a couple photos of the microburst from the same storm.

**Comment by Meso22** [comment]
![image](https://user-images.githubusercontent.com/105577122/206810529-cf6b4f00-f796-4a88-948e-1b171ef7d16f.png)
Here is another at sunrise

**Reply by MET905** [reply]
Wow that's a lot of energy at sunrise, Was that triggered by a front?

**Reply by Meso22** [reply]
Very high moisture over the ocean and 25 degree temp causing strong convection over the flat area.

**Reply by Meso22** [reply]
We usually get this off the east coast of australia during winter when the temps get down to 0 in the early mornings

**Reply by Meso22** [reply]
The ocean is also pretty warm during winter

**Comment by MET905** [comment]
Another Arctic air outbreak has set the stage for another Lake Effect Snowstorm. this time lake effects bands are not as organized due to high levels of shear in the atmosphere. This area will be getting a break from snow as high pressure is slowly building in and warm unseasonable warm temps are expected.  

![lake effect snow setup](https://user-images.githubusercontent.com/113155608/207759828-0449fcd9-e426-44ee-8783-85cd1a5d570a.png)

**Reply by Meso22** [reply]
Very cool!

**Comment by Meso22** [comment]
![image](https://user-images.githubusercontent.com/105577122/207988957-04365759-cb78-4095-992e-0bff64827947.png)
I managed to create a large storm even from one day of moisture and 25 degree temps

**Comment by okienative99** [comment]
I created a supercell thunderstorm in a warm, favorable environment with high CAPE values and dewpoints in the upper 60s. The cold front created cool dry air which crashed with the warm moist air. Still playing around with the simulation, but would like to study why thunderstorms increase in intensity when moving over a mountain range. 
![storm2](https://user-images.githubusercontent.com/120753895/208195688-fe332b80-b678-4967-aadb-0a402d0c1fc3.JPG)

**Reply by JustLearninglol** [reply]
They increase in intensity due to orographic lift. If wind is coming directly at a mountain range, it is forced to rise, then it cools, condenses, and then precipitation! That can boost thunderstorms intensity to become heavy!!! Once the storm passes the mountain range, the winds go downslope instead and can kill the storm. That is called a rain shadow!

**Reply by okienative99** [reply]
@JustLearninglol Thanks for telling me! I used to live in SW Oklahoma (I now live in Oklahoma City - the heart of tornado alley.) where the Wichita Mountain range is located. An interstate named the H.E. Bailey Turnpike runs northeast through the entire state, and tornadoes love to parallel this turnpike. I have been doing some research, and I love this simulation because I believe the mountains serve to enhance storms when moving toward Oklahoma City. This simulation has taught me the dynamics of weather when mimicking the landscape of Oklahoma. The city experiences tornadoes fairly often, and there is an ongoing study to see if this is what's causing tornadoes to form in our area!

**Comment by Meso22** [comment]
![image](https://user-images.githubusercontent.com/105577122/208270850-b2a87037-2c58-4473-9c98-690045490ca0.png)
A very large late afternoon supercell

**Reply by okienative99** [reply]
@MET905 at what settings were you able to achieve this? This looks amazing!

**Reply by Meso22** [reply]
I have most of the map ocean and some of it land I have the water temp set to 35c and the sun intensity full. Also set the wind to full and the drag to 0.1

**Comment by MET905** [comment]
High pressure in place with beautiful temperatures. Cold air is on the way again... Id say a wind storm is going to happen because of the temp gradient! 
![high](https://user-images.githubusercontent.com/113155608/208271689-20a95b0f-a9cd-4874-bcb1-7949896bd92d.png)

**Reply by Meso22** [reply]
I would love to see it if it does happen!

**Reply by MET905** [reply]
its Going to be epic! I'm expecting a nasty frontal Snow Squall mixed with rain and damaging winds ahead of the front. also with the amount of moisture available I expect a brief burst of intense snow as there will be dynamic cooling. the question is how long will that shot of cold air last before it turns back into rain as warm air will likely overtake the cold front from the insane gradient. besides the frontal precipitation I expect a very nasty burst of intense lake effect snow/rain to develop. HOWEVER with how crazy the air will be moving I don't think lake effect snow development will be fully supported. BUT if that cold air wins the battle and it sits over the warm lake and the air has a chance to settle down with a somewhat gentle wind over Lake Erie.... I think a major snow squall event will be possible at the lee of the lake, Its so much fun trying to forecast whats going to happen in this sim lol!

**Comment by Meso22** [comment]
![image](https://user-images.githubusercontent.com/105577122/208280255-48fb2a93-90e4-49b2-9ad0-9d12895c01f0.png)
A small thunderstorm approaching the city.

**Comment by MET905** [comment]
Niels must love seeing us weather nuts enjoy his sim! Iv been looking for something like this for years!

**Reply by Meso22** [reply]
About this time last year I found the weathersim and I have been playing it since.
I was looking for cumulonimbus simulators and weather simulators when I came across it. It was exactly what I was looking for.

**Comment by Meso22** [comment]
![image](https://user-images.githubusercontent.com/105577122/208283002-a4ac0034-7075-4194-9ccd-a9613287e302.png)
A scorching 40 degree low pressure day
I am going to get some moisture to kick in and see what happens

**Reply by JustLearninglol** [reply]
I can remember a sim I made which was a sim of the Sahara desert where daily temps were 45-47c. It was extremely hot daily, and gusty winds would develop a little past noon. I remember a time when it even topped 50c one day. I don't have the world, because I never saved it :P, but I added a little moisture one day, and some powerful dry thunderstorms occured causing haboobs. Pretty neat

**Reply by Meso22** [reply]
wow 50c that's crazy!

**Comment by JustLearninglol** [comment]
<img width="1440" alt="Screen Shot 2022-12-17 at 6 40 17 PM" src="https://user-images.githubusercontent.com/105594008/208283917-d3ef3ff7-15b7-42d6-bf2e-0e4970a5cf7f.png">
Heres a small little 1250x300 world that I made since my computer isn't very fast. This is a great example of orographic lift that can occur. I made a large flat landmass next to a warm ocean, with standard evaporation parameters. During the days, the coastline gets windy, and the winds come inland and eventually get forced upward to form pretty massive downpours over the mountains. The downpours cause downsloping winds on the otherside of the mountains, and then cause more winds to come in again from the coast to the flatlands, and the cycle continues. This is fun as it can sometimes create excessive rainfall over the mountains. I don't know how to share the download.

**Reply by Meso22** [reply]
Wow very nice!

**Reply by okienative99** [reply]
@JustLearninglol a very great example! Thanks for sharing.

**Comment by Meso22** [comment]
![image](https://user-images.githubusercontent.com/105577122/208283812-4d077bbc-c742-46bc-b35b-f9b030124fc5.png)
![image](https://user-images.githubusercontent.com/105577122/208283945-295a4b86-fd8f-49ac-b574-180aefdca8b1.png)
![image](https://user-images.githubusercontent.com/105577122/208284018-311c620d-81f4-4af8-be36-8303ba7a928b.png)
![image](https://user-images.githubusercontent.com/105577122/208284098-5fe7af5f-a1c2-4921-a859-31d71630e367.png)
![image](https://user-images.githubusercontent.com/105577122/208284232-e737ebda-5b83-422c-8e8b-bc77971324aa.png)
![image](https://user-images.githubusercontent.com/105577122/208284265-1e456303-5437-4d1e-b9d5-2349721ed7c0.png)
Here is a GIGA collection of very severe storms I created after a week of 40c heat!

**Reply by okienative99** [reply]
@Meso22 WOW! These look just like the spring storms that form here at home!

**Reply by Meso22** [reply]
Yes spring thunderstorms always have a lot of heavy moisture and tend to be very big!

**Comment by panagiotisLadis** [comment]
![image](https://user-images.githubusercontent.com/60401937/208291901-d5378a27-2d85-4d13-bdb1-8524113d20ac.png)


**Snowwwwwwwwwwwwwwwwwww!!!!!!!**

**Reply by Meso22** [reply]
Very Awesome!

**Comment by Meso22** [comment]
![image](https://user-images.githubusercontent.com/105577122/208326728-bac2858e-25f7-4f43-8e32-e801f3ceb654.png)
A storm from the new update

**Reply by niels747** [reply]
The top of the clouds looks so bright! Have you turned the sunintensity or the exposure up?

**Reply by Meso22** [reply]
Yes I usually turn the sun intensity up so I can get more heat

**Comment by GRolLevente** [comment]
Thank you very much for the update! The vector rendering and the movement of the trees by the wind is a great idea. The image shows a storm system forming over the sea as it reaches land.

Link to the simulation: https://drive.google.com/drive/folders/1DhuKpMHR6yXHO0IYGE7weBLjtmCCcYlm?usp=sharing

![Test_sim1](https://user-images.githubusercontent.com/118711061/208414399-4b61fd41-7f03-4826-8784-dcd7f152612d.png)

**Comment by Meso22** [comment]
![image](https://user-images.githubusercontent.com/105577122/208425034-01ed9b65-b950-49aa-8782-69096431da88.png)
Shadow Curvature?

**Reply by JustLearninglol** [reply]
These crazy things happen during sunset

**Reply by niels747** [reply]
That happens because light doesn't move instantly in the sim and while it's traveling the angle changes,

**Reply by Meso22** [reply]
I love it! very realistic!

**Comment by Meso22** [comment]
![image](https://user-images.githubusercontent.com/105577122/208431741-1152497c-af1b-4701-983f-2705734ed59e.png)
A waterspout ps: I know tornadoes/waterspouts cannot form on the sim because it's 2d but I like to pretend anyways

**Reply by okienative99** [reply]
@Meso22 very photogenic!

**Reply by Meso22** [reply]
Thanks!

**Reply by niels747** [reply]
Yea that's basically a very narrow and strong updraft. If you turn the vorticity up to 0.010 it would be to unstable wich is more realistic.

**Reply by Meso22** [reply]
I never knew that higher vorticity was more realistic!

**Comment by MET905** [comment]
![COUDS](https://user-images.githubusercontent.com/113155608/209236730-2437f112-253b-4ca1-8cdb-e8d81fe01c0a.png)
Clouds behind the warm front. you can see the wind shear taking place as the cold front is advancing.

**Comment by Meso22** [comment]
![image](https://user-images.githubusercontent.com/105577122/209301764-4040967b-6708-4c4d-8716-34143032ffac.png)
A 12km bushfire

**Reply by Meso22** [reply]
the fire was moving at 6km an hour (3mph)

**Reply by niels747** [reply]
Did you measure the distance by taking samples and subtracting the x coordinates to calculate the number of horizontal cells and then multiplying that by the size of a cell?

**Reply by Meso22** [reply]
I just put the mouse cursor at the front of the fire and waited an hour (ingame) and then measured the distance the fire travelled in the hour by using the sounding graph to get different height measurements and testing the different heights to see if they match the distance between the front of the fire and where I put the cursor.

**Comment by Meso22** [comment]
https://www.youtube.com/watch?v=VkDFs8ig87M
My Tropics Timelapse vid

**Reply by Meso22** [reply]
save https://drive.google.com/file/d/1PTCki7UO8rWbZx7qgCts94t_jYcLgHLA/view?usp=sharing

**Comment by MET905** [comment]
![snowsquall](https://user-images.githubusercontent.com/113155608/209744012-650b6b7f-f982-4fd1-80d9-521f4f28346f.jpg)
Snowsquall developing over Lake Erie behind a cold front!

**Comment by kimswx** [comment]
![storm3](https://user-images.githubusercontent.com/121654053/210043652-bdba90f8-e6b5-42b4-9764-62504ee0ecf9.PNG)
![storm2](https://user-images.githubusercontent.com/121654053/210043654-395a6939-e24f-44b9-9bfc-0171dc56f4c0.PNG)
Some summer storms forming after a warm afternoon.

**Reply by Meso22** [reply]
nice

**Comment by JustLearninglol** [comment]
This is my island world! The simulation is around 80% ocean and 20% land, so there is lots of ocean! Here is a day in the island!
![Screen Shot 2022-12-30 at 7 22 42 PM](https://user-images.githubusercontent.com/105594008/210123545-680b1b19-7d92-4cb1-88f8-642756e321f5.png)
It was a comfortable morning for the entire island with warm temps near the sea and the valley. The mountains were pretty chilly, as they soar pretty high! 
![Screen Shot 2022-12-30 at 7 25 37 PM](https://user-images.githubusercontent.com/105594008/210123588-3b17ebaa-35c8-46d6-b33e-4e7d72ce110d.png)

As soon as it was Noon, winds coming from the right caused orographic lift, forming a steady rain over the mountains! They soar up to 7800ft! It is always cold up there! The right side coastal area remained clear with temps in the comfy 70s!
![Screen Shot 2022-12-30 at 7 26 53 PM](https://user-images.githubusercontent.com/105594008/210123602-93b71c11-01f7-4cdc-805e-6545e82df1d8.png)
As afternoon progressed, the left side cleared up, but as soon as that happened, new storms forming on the right side spoiled the sunny day they were having! The left side finally had time to bathe in sunshine for a while..
![Screen Shot 2022-12-30 at 7 27 58 PM](https://user-images.githubusercontent.com/105594008/210123617-4551a5d7-684a-4eed-8e8a-bc685fa7ba8d.png)
The right side finally got sunny again quickly, while the storms moved to the left, powered by orographic lift and intensifying to drop heavy steady rain over the mountains once more.
![Screen Shot 2022-12-30 at 7 28 56 PM](https://user-images.githubusercontent.com/105594008/210123630-815f01fa-3695-4d4e-ae31-092043808e16.png)
As sunset neared, the left side cleared, and boom, as it clears, as usual, the right side got stormy again with a few showers. The top got some snow! 
![Screen Shot 2022-12-30 at 7 30 28 PM](https://user-images.githubusercontent.com/105594008/210123647-9a688b3c-193c-4b6f-b7f9-da772ef7b4d4.png)
As the sun was setting, widespread steady rain developed and covered the island with raindrops! What a lovely evening! 
![Screen Shot 2022-12-30 at 7 32 13 PM](https://user-images.githubusercontent.com/105594008/210123666-61b38fcc-e08a-4125-b01d-d6d282020dc3.png)
As night began, at 9 PM ingame, the rain eventually broke down, and clear skies developed! I expect, if the skies remain clear, for some radiation fog at the valley. Temps cooled down, and a chilly night is beginning. this is the day in an island!

**Comment by kimswx** [comment]
![storm6](https://user-images.githubusercontent.com/121654053/210125005-48650e7b-778e-4795-9d18-e5ad27813323.PNG)
A Storm had been spotted forming behind an advancing cold front. The local isles had been happy after they finally recorded rainfall after days of dry and pretty hot weather, Donh Isles (The small island) had recorded the coldest temperature that month of 8C, during the time the front advances the temperature there roughly decreases by up to 10C!

**Reply by kimswx** [reply]
Update: Donh Isles records an even colder temperature of about 7C

**Comment by odvigajlo** [comment]
![s2](https://user-images.githubusercontent.com/103889327/210174574-cc22f6a3-cb40-4415-b2d7-1ffcbd9ee17a.png)
Kinda cool looking mountain convection

**Reply by odvigajlo** [reply]
![s2](https://user-images.githubusercontent.com/103889327/210174644-5ccd2702-9442-4b29-8436-b97e8153fd55.png)
Quite Powerful Convection

**Comment by Meso22** [comment]
![image](https://user-images.githubusercontent.com/105577122/210188380-fc7be2b5-1263-477f-b736-e42061656966.png)
9960m (32800ft) Cumulonimbus

**Reply by Meso22** [reply]
The anvil reached 11000m (37000ft)

**Comment by Meso22** [comment]
![image](https://user-images.githubusercontent.com/105577122/210191856-a6bd22f4-1f3e-4bc8-8942-1555298f9d74.png)
A anvil reached the very top of the map and was half the size of a 6000 horizontal resolution map!

**Comment by MET905** [comment]
![wind](https://user-images.githubusercontent.com/113155608/212230947-0bb9bcaf-60b9-4686-aff3-4b38e24ad553.png)
Crazy wind speeds recorded with this winter storm!

**Reply by niels747** [reply]
That sure is insane, what did you do?

**Reply by MET905** [reply]
I had really cold temperatures and increased the global heating causing a Temperature gradient wind storm

**Comment by MET905** [comment]
![wet](https://user-images.githubusercontent.com/113155608/213545331-ac9d7a01-5544-49dc-820d-adf6dbf15859.png)
Very moist air has been building over the area for days due to a strong flow off of the golf of Mexico,
Torrential downpours and heavy cloud cover have been the result, I'm sure if the temperatures were colder this would be a major snowstorm.

**Comment by okienative99** [comment]
![c1](https://user-images.githubusercontent.com/120753895/218281378-19216b8a-e0e3-44ad-b162-f81d8c80a350.JPG)
![c2](https://user-images.githubusercontent.com/120753895/218281379-b3f922a0-06fc-4597-ad20-a30336bd9a64.JPG)
Huge high precipitation thunderstorm! Temperatures in the low 80s with straight line winds around 100 mph.

**Reply by MET905** [reply]
Ahh the yellow snow lol!

**Comment by MET905** [comment]
![STORM](https://user-images.githubusercontent.com/113155608/218918249-a60e60bf-705e-42bd-aca8-9216976ea9f0.png)
I think its safe to say I created a moderate Thunderstorm in the middle of Feb, warm abnormal temperatures with an advancing cold front. after the front passed temperatures crashed to -15 from 20 degrees! this sharp gradient also created some damaging winds and quick frontal snowsqualls before high pressure took over.

**Comment by raincloud3** [comment]
![big stormclouds](https://user-images.githubusercontent.com/123894139/219160388-d3431ca9-b232-4bc6-a766-6d48dd0a3fd8.png)
very intense storm

**Comment by raincloud3** [comment]
![image](https://user-images.githubusercontent.com/123894139/220342549-20a38a30-4e4d-4c04-9cb2-890c13461189.png)
210 km/h windspeed during storm

**Comment by MET905** [comment]
![STORM](https://user-images.githubusercontent.com/113155608/220793543-4e0df8e4-f96d-4a9d-9396-7f0a69d1e2e0.png)
The Ice storm has begun! major Freezing rain event unfolding.

**Comment by okienative99** [comment]
![s3](https://user-images.githubusercontent.com/120753895/225129235-bc36b9d0-5600-43e5-8d35-747a003362de.JPG)
The day started with clear skies and air temps in the 60s. The dew point was at 60 degrees, perfect for severe weather.

**Reply by okienative99** [reply]
![s5](https://user-images.githubusercontent.com/120753895/225129784-8a832eaa-c490-432d-a7dd-a765377d73af.JPG)
By noon, the dew point temps increased as moisture surged around 20-25 MPH to the north. This started to cause some partly cloudy skies over the plains. The surface temperature started to increase into the low 70s.

**Reply by okienative99** [reply]
![s7](https://user-images.githubusercontent.com/120753895/225130410-c7793dda-66f3-4145-80b8-f9b4d4af382f.JPG)
The first storm of the day fired off to the north of the city, causing hail and heavy rain over the foothills.

**Reply by okienative99** [reply]
![s8](https://user-images.githubusercontent.com/120753895/225130779-dd28ab0e-baa9-42e3-ac20-25b3fb48db23.JPG)
In the afternoon, towers began to form over the city area.

**Reply by okienative99** [reply]
![s9](https://user-images.githubusercontent.com/120753895/225131090-753c9028-c3cf-4176-b27f-986afc6a5b7b.JPG)
A huge supercell thunderstorm formed over the city producing damaging winds up to 80-90 mph as it quickly raced off to the north at 30 mph. The storm kicked up dust and produced lots of rain and hail.

**Reply by okienative99** [reply]
![s10](https://user-images.githubusercontent.com/120753895/225131973-c1b3a207-1916-405a-9003-051c66abaadc.JPG)
After an active day of storms, things got quiet during the evening. A beautiful sunset took place with leftover anvil from thunderstorms.

**Reply by coolkid0281** [reply]
Love it

**Comment by MET905** [comment]
![BREWING](https://user-images.githubusercontent.com/113155608/230182031-3c5b66ef-bf5d-421b-9770-dfd24d6b397f.png)
Very nice and sunny currently, Expecting thunderstorms by 3-4 pm as daytime heating increases

**Comment by hectorization** [comment]
![image](https://user-images.githubusercontent.com/130441928/231336903-d782c08a-8d05-4dd4-8e1d-420d8673463e.png)
a large storm

**Comment by okienative99** [comment]
Classic supercell thunderstorms today. 
![st1](https://user-images.githubusercontent.com/120753895/232884468-82189416-1071-414c-b4bc-d4ec1e9298d8.JPG)
![st2](https://user-images.githubusercontent.com/120753895/232884472-5045a67a-f7b7-46a7-b0de-72ac01baaec5.JPG)
![st3](https://user-images.githubusercontent.com/120753895/232884473-c12d0681-e15c-4ce5-8a4a-e38ab7280c4c.JPG)
![st4](https://user-images.githubusercontent.com/120753895/232884476-9a33f37f-bceb-4678-9ec3-60fcbee7a0f4.JPG)

**Comment by MET905** [comment]
![STORM](https://user-images.githubusercontent.com/113155608/235581575-4ac66d9c-fa0d-423d-8ca9-105efd8f0bad.png)
after a full week of sunshine and warm temperatures a Cold front moving over the area is producing thunderstorms that are producing incredible rainfall amounts. the cold front has LOTS of instability to work with so i expect this deep convection to continue into the night before finally ramping down and turning into isolated showers.

**Comment by qfmp1012** [comment]
![Screenshot (5)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/65e2cadd-fa99-40b5-b7d7-9f4b93b00378)
I do not know to disable the top bar but W or L?

**Reply by qfmp1012** [reply]
![Screenshot (6)](https://github.com/niels747/2D-Weather-Sandbox/assets/135496047/3271b7fc-529f-441d-8e1e-57ff211d98de)
It formed mammatus clouds

**Comment by coolkid0281** [comment]
Uh guys I was waiting for like a month in game I got a CRAZY COLD FRONT I did not get pics but in front of the cold front had 140 mph winds at some places and crazy storms with hail and 120 mph line of storms. Here is the only pic i got 
![image](https://github.com/niels747/2D-Weather-Sandbox/assets/110744575/bfc84c6b-6a66-415d-a251-1b40b6030b35)

**Comment by coolkid0281** [comment]
![image](https://github.com/niels747/2D-Weather-Sandbox/assets/110744575/18407116-5f0c-4e7c-b92d-627aa5cd9b3b)
here is the start of the day I think it's going to be a sunny day with a low chance of thunderstorms due to the low amount of cape
